src/main/java/codechicken/nei/api/API.java

```diff
@@ -15,21 +15,15 @@
 import codechicken.nei.api.ItemFilter.ItemFilterProvider;
 import codechicken.nei.config.Option;
 import codechicken.nei.config.OptionKeyBind;
-import codechicken.nei.recipe.GuiCraftingRecipe;
-import codechicken.nei.recipe.GuiUsageRecipe;
-import codechicken.nei.recipe.ICraftingHandler;
-import codechicken.nei.recipe.IUsageHandler;
-import codechicken.nei.recipe.RecipeInfo;
-import codechicken.nei.recipe.StackInfo;
+import codechicken.nei.recipe.*;
```

Comment:
I personally prefer explicit imports and avoiding the `.*`  Please configure your IDE to either add explicit imports or to not change existing ones.